docs/src/main/sphinx/sql.md

```diff
@@ -78,3 +78,9 @@ sql/update
 sql/use
 sql/values
 ```
+
+```{toctree}
+:hidden:
+
+sql/pattern-recognition-in-window
```

Comment:
maybe we should also hide MATCH-RECOGNIZE from the toc following the logic that all statmenents in the nav are top level executable